### PR TITLE
Add more clients

### PR DESCRIPTION
**Pull request by sjehuda**
aTalk, blabber.im, BombusMod, Cheogram, Kontalk, Moxxy, Quicksy and Stork IM.

### CODE REVIEW

**Comment by sjehuda** [comment]
Consider:

Conv6ations
https://dev.sum7.eu/sum7/Conversations

Snikket
https://snikket.org/

See #27

File: clients_Android.json [hunk 1]
```diff
@@ -1,7 +1,15 @@
 [
-    "<a href=\"https://play.google.com/store/apps/details?id=org.yaxim.androidclient\"><img src=\"assets/yaxim.svg\" height=\"60\">yaxim</a>",
+    "<a href=\"https://play.google.com/store/apps/details?id=org.atalk.android\"><img src=\"assets/atalk.png\" height=\"60\">aTalk</a>",
```
Comment (licaon-kter) [inline clients_Android.json:2]: https://f-droid.org/en/packages/org.atalk.android/ ?

File: clients_Android.json [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+    "<a href=\"https://f-droid.org/en/packages/de.pixart.messenger/\"><img src=\"assets/blabber.png\" height=\"60\">blabber.im</a>",
+    "<a href=\"https://play.google.com/store/apps/details?id=org.yaxim.bruno\"><img src=\"assets/bruno.png\" height=\"60\">Bruno</a>",
+    "<a href=\"https://play.google.com/store/apps/details?id=org.bombusmod\"><img src=\"assets/bombusmod.png\" height=\"60\">BombusMod</a>",
+    "<a href=\"https://f-droid.org/en/packages/com.cheogram.android/\"><img src=\"assets/cheogram.png\" height=\"60\">Cheogram</a>",
     "<a href=\"https://play.google.com/store/apps/details?id=eu.siacs.conversations\"><img src=\"assets/conversations.svg\" height=\"60\">Conversations</a>",
+    "<a href=\"https://f-droid.org/packages/org.kontalk/\"><img src=\"assets/kontalk.png\" height=\"60\">Kontalk</a>",
```
Comment (licaon-kter) [inline clients_Android.json:8]: I Kontalk even developed anymore? 2020? https://github.com/kontalk/androidclient/releases

**Comment by licaon-kter** [comment]
> Conv6ations

No longer developed

> Snikket

Cheogram might be better since it's on par with upstream

File: clients_Android.json (hunk 2 above)
Comment (sjehuda) [reply clients_Android.json:8]: > Is Kontalk even developed anymore? 2020?

I don't know.

@kontalk

**Comment by pravi** [comment]
As mentioned in #27 snikket needs an invite or an existing xmpp account.

image

Only forks of conversations has good compliance on android. aTalk, BombusMod and Moxxy we will have to collect compliance data before we recommend. Quicksy has same compliance as conversations, as both are built from the same source code.

**Comment by pravi** [comment]
For aTalk there is a request for collecting compliance data https://github.com/cmeng-git/atalk-android/issues/197

**Comment by ge0rg** [comment]
This is about ease of use for newcomers, so Google Play is the way to go. If a user can install F-Droid, they will figure out installing an XMPP client without this landing page. Also we are only recommending clients that are actually well-maintained and compliant.

**Comment by sjehuda** [comment]
I think Moxxy is a good addition to the list.

**Comment by ge0rg** [comment]
> I think Moxxy is a good addition to the list.

I never heard of it, and it's not on Google Play.

**Comment by sjehuda** [comment]
Indeed.

It's on IzzyOnDroid F-Droid Repository
https://apt.izzysoft.de/fdroid/index/apk/org.moxxy.moxxyv2

Repository
https://codeberg.org/moxxy/moxxyv2

Homepage
https://moxxy.org/

From @PapaTutuWawa

**Comment by licaon-kter** [comment]
> Indeed. It's on IzzyOnDroid F-Droid Repository Repository https://codeberg.org/moxxy/moxxyv2 

if you read the release notes you'll get a clearer picture of how far away still is https://codeberg.org/moxxy/moxxy/releases

**Comment by sjehuda** [comment]
It's fine.

We now know of Moxxy.

The Moxxy developer now knows of Modern XMPP and easy-xmpp-invitation.

I guess Moxxy will be featured here next year if not later this year.

Patience...

**Comment by licaon-kter** [comment]
> This is about ease of use for newcomers, so Google Play is the way to go.

Newcomers will see _"you need to pay to Play"_ and they'll go-a-way, easily.

**Comment by sjehuda** [comment]
> Newcomers will see "you need to pay to Play" and they'll go-a-way, easily.

Good point.

I suggest to add a guide or a link to a guide to install F-Droid with a short "intro" about its benefits.

**Comment by licaon-kter** [comment]
A simple _"These apps might be available on F-Droid as well"_ or smth

**Comment by sjehuda** [comment]
These apps are also available on F-Droid ([What is F-Droid?]())

Instead of a link, we can make it to be a tooltip which will pop div with a short paragraph which will be displayed upon hover.

**Comment by sjehuda** [comment]
A greater issue is the tendency of "Play" store to (1) ban, (2) block, (3) censor, (4) hide and (5) remove software, and also (6) rename URL addresses.

# Case 1
## Conversations

> To celebrate Conversations being back on the Play Store and to deny Google their 15% cut I made the app free for the next ~48 hours. 😜

> If you are new here: Check out [#XMPP](https://gultsch.social/tags/XMPP). It's federated, provider and vendor independent instant messaging. It’s great!

> Or get it from [#fdroid](https://gultsch.social/tags/fdroid) - That version even has address book integration and public channel discovery!  😂

https://gultsch.social/@daniel/111933922710829462

I do wonder why F-Droid version also offers "address book integration and public channel discovery" and the one on "Play" does not.

I suppose it is because _they_ do not want XMPP to appeal "normal people".

# Case 2
## monocles

> URL for monocles chat in Play Store has changed.

https://github.com/modernxmpp/easy-xmpp-invitation/pull/65

@p42ity

Do the people working on "Play" server not know how to use URL redirection?

# Case 3
## blabber.im

> Nachdem Google den blabber.im Messenger als Spyware eingestuft und aus dem Playstore gelöscht hat, wird die Playstore Version künftig neben der Git Version nur noch über Codeberg zum Download angeboten. Der interne Updater wird ab Version 3.1.2 die jeweils richtig Version anzeigen und herunterladen. Bei F-Droid bleibt bisher alles wie gehabt.

https://kanoa.de/@blabber/108289026302640506

---

By linking to "Play" store, we are harming ourselves in both the short and long terms.

I urge you and highly recommend you to drop *any* link to that "Play" catalogue!

If a server admin wants to add links that are not from F-Droid, then it is possible by modifying the source files.

**Comment by sjehuda** [comment]
A greater issue is the tendency of "Play" store to (1) ban, (2) block, (3) censor, (4) hide and (5) remove software, and also (6) rename URL addresses of software related to XMPP.

## Case 1
## Conversations

> To celebrate Conversations being back on the Play Store and to deny Google their 15% cut I made the app free for the next ~48 hours. 😜
>
> If you are new here: Check out [#XMPP](https://gultsch.social/tags/XMPP). It's federated, provider and vendor independent instant messaging. It’s great!
>
> Or get it from [#fdroid](https://gultsch.social/tags/fdroid) - That version even has address book integration and public channel discovery!  😂

https://gultsch.social/@daniel/111933922710829462

I do wonder why F-Droid version also offers "address book integration and public channel discovery" and the one on "Play" does not.

I suppose it is because _they_ do not want XMPP to appeal "normal people".

## Case 2
## monocles

> URL for monocles chat in Play Store has changed.

See https://github.com/modernxmpp/easy-xmpp-invitation/pull/65 from @p42ity

Do the people working on "Play" server not know how to use URL redirection?

## Case 3
## blabber.im

> Nachdem Google den blabber.im Messenger als Spyware eingestuft und aus dem Playstore gelöscht hat, wird die Playstore Version künftig neben der Git Version nur noch über Codeberg zum Download angeboten. Der interne Updater wird ab Version 3.1.2 die jeweils richtig Version anzeigen und herunterladen. Bei F-Droid bleibt bisher alles wie gehabt.

https://kanoa.de/@blabber/108289026302640506

[blabber.im](https://blabber.im/) is not a spyware, and is also listed at [F-Droid](https://f-droid.org/packages/de.pixart.messenger/).

---

By linking to "Play" store, we are harming ourselves in both the short and long terms.

I urge you and highly recommend you to drop *any* link to that "Play" catalogue!

If a server admin wants to add links that are not from F-Droid, then it is possible by modifying the source files.

It would best wiser - to all of us - to provide a default selection from the F-Droid catalogue which has no spywares/trackers and does not load over 1MB to 10MB for each page load.

**Pull request closed without merging**